easybuild/tools/job/gc3pie.py

```diff
@@ -96,9 +93,19 @@ def init(self):
 
         Start a new list of submitted jobs.
         """
-        self.output_dir = os.path.join(os.getcwd(), 'easybuild-gc3pie-jobs')
+        cfgfile = build_option('job_backend_config')
+        if cfgfile:
+            # FIXME: is there a better way?
+            gc3libs.Default.CONFIG_FILE_LOCATIONS.append(cfgfile)
+
+        # additional subdirectory, since GC3Pie cleans up the output dir?!
```

Comment:
is this correct @riccardomurri?

If so: please change that (I ended up removed my entire `easybuild-framework` directory without realising it...).